README.md

```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
 **Release Date**: October 27, 2021
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/2.12.0).
-
----
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/protocol_spec/releases/tag/5.4.1).
 
-#### Manticore: 2.9.0
-**Release Date**: December 8, 2021
+--
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/manticore/releases/tag/2.9.0). 
+#### SDL RPC Spec: 8.0.0
+**Release Date**: October 27, 2021
 
----
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/rpc_spec/releases/tag/8.0.0).
 
 ## Upcoming Releases
 
+#### SDL Core: 8.2.0
 
-#### SDL iOS: 7.4.0
-**Target Release Date**: April 13, 2022
+**Target Release Date**: October 26, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/27).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/22).
 
 ---
 
-#### SDL Java Suite: 5.4.0
-**Target Release Date**: April 13, 2022
+#### SDL iOS: 7.3.3
+**Target Release Date**: June 29, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_java_suite/projects/21).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/29).
 
 ---
 
-#### SDL JavaScript Suite: 1.6.0
-**Target Release Date**: April 13, 2022
+#### SDL Java Suite: 5.5.0
+**Target Release Date**: June 29, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_javascript_suite/projects/7).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_java_suite/projects/23).
 
 ---
 
-#### SDL Protocol Spec: 5.5
-**Target Release Date**: April 13, 2022
+#### SDL JavaScript Suite: 1.7.0
+**Target Release Date**: June 29, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/protocol_spec/projects/7).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_javascript_suite/projects/8).
 
 ---
 
-#### SDL RPC Spec: 8.1.0
-**Target Release Date**: April 13, 2022
-
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/rpc_spec/projects/8).
+#### SDL Policy Server: 3.1.0
+**Target Release Date**: October 26, 2022
 
----
-#### SDL Core: 8.1.0
-**Target Release Date**: April 13, 2022
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_server/projects/8).
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/21).
+--
 
----
-#### SDL Policy Server: 2.13.0
-**Target Release Date**: April 13, 2022
+#### SDL Protocol Spec: 5.4.2
```

Comment:
```suggestion
#### SDL Protocol Spec: 5.5.0
```

We should plan for this to be a minor release as usual, and will update accordingly if we determine only a hotfix is needed.